rules/inclusive-gateway.js

```diff
@@ -0,0 +1,52 @@
+const { is } = require('bpmnlint-utils');
+
+const { hasProperties, ERROR_TYPES } = require('./utils/element');
+
+const { reportErrors } = require('./utils/reporter');
+
+module.exports = function() {
+  function check(node, reporter) {
+    if (!is(node, 'bpmn:InclusiveGateway')) {
+      return;
+    }
+
+    const incoming = node.get('incoming');
+
+    if (incoming && incoming.length > 1) {
+      const error = {
+        message: `Element of type <${ node.$type }> must have one property <incoming> of type <bpmn:SequenceFlow>`,
```

Comment:
Shall we make it human-friendly?

```suggestion
        message: `Merging <bpmn:inclusiveGateway> not allowed`,
```

Reply:
This is a technical error message that is adjusted in https://github.com/camunda/linting/pull/18. It is about what the error is on a technical level. That this is about merging is more meta. For consistency I would keep it as it is. The error message for extension elements is `Element of type <...> must have one extension element of type <...>`.

Reply:
Well, if it's adjusted, let's keep it as is.